src/app.js

```diff
@@ -138,6 +140,47 @@ module.exports.run = function (worker) {
     // libdata: res.locals.libdata
   };
 
+  const getContext = function(req, res, next) {
+    if (!USE_SMAUG) {
+      req.context = dummyContext;
+      return next();
+    }
+
+    var authHeader = req.get('authorization');
+    if (typeof authHeader === 'undefined') {
+      return next();
+    }
+
+    var authType = authHeader.split(' ', 2)[0];
+    var bearerToken = authHeader.split(' ', 2)[1];
```

Comment:
Another more ES6-like approach could probably be `let [authType, bearerToken] = authHeader.split(' ', 2);`, but this works fine